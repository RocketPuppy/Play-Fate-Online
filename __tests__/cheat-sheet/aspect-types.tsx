import * as React from "react";
import * as renderer from "react-test-renderer";
import AspectTypes from "../../src/cheat-sheet/aspect-types";

test("renders all the pieces", () => {
  const render = renderer.create(<AspectTypes />);

  expect(render.toJSON()).toMatchSnapshot();
});
