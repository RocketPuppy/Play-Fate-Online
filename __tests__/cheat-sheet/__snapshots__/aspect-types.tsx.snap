// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders all the pieces 1`] = `
<div>
  <h2>
    Aspect Types
  </h2>
  <dl>
    <dt>
      Game aspects
    </dt>
    <dd>
      permanent, made during game creation
    </dd>
    <dt>
      Character aspects
    </dt>
    <dd>
      permanent, made during character creation
    </dd>
    <dt>
      Situation aspects
    </dt>
    <dd>
      last for a scene, until overcome, or until irrelevant
    </dd>
    <dt>
      Boosts
    </dt>
    <dd>
      last until invoked one time
    </dd>
    <dt>
      Consequences
    </dt>
    <dd>
      last until recovered
    </dd>
  </dl>
</div>
`;
